src/main/java/bo/SearchActions.java

```diff
@@ -18,17 +18,17 @@
 
     @Step("open website : {url}")
     public void openWebSite() {
```

Comment:
you have the same method in CartActions class

Reply:
Better extract this method to some other class
Create some class 
for example CommonActions

Reply:
fixed